extensions/BMO/template/en/default/account/create.html.tmpl

```diff
@@ -185,5 +210,4 @@ function onSubmit() {
     <div>[% desc FILTER html %]</div>
   </td>
   </tr>
-[% END %]
-
+[% END %e]
```

Comment:
Typo - this won't compile. Are you testing these changes on a local install of Bugzilla?